examples/shared_samples/enclosure.rb

```diff
@@ -80,6 +82,51 @@
 item2.retrieve!
 puts "\nUpdated #{type} with new name = '#{item2[:name]}' successfully.\n  uri = '#{item2[:uri]}'"
 
+if @client.api_version >= 600
+  # Gets a enclosure by scopeUris
+  scope_class = OneviewSDK.resource_named('Scope', @client.api_version)
+  scope_1 = scope_class.new(@client, name: 'Scope 1')
+  scope_1.create
+  query = {
+    scopeUris: scope_1['uri']
+  }
+  puts "\nGets a logical enclosure with scope '#{query[:scopeUris]}'"
+  item4 = encl_group_class.get_all_with_query(@client, query)
```

Comment:
is this working? Because we need to add this scope in enclosures to fetch enclosures.